"ably/promises" fails with `Cannot find module './nodejs/index'`

At least with v1.2.7, importing `ably/promises` fails with `Cannot find module './nodejs/index'` because it does not include `nodejs/` directory.

To reproduce, install the latest `ably`.

```text
❯ tree --dirsfirst
.
├── browser
│   └── static
│       ├── ably-commonjs.js
│       ├── ably-commonjs.noencryption.js
│       ├── ably.js
│       ├── ably.min.js
│       ├── ably-nativescript.js
│       ├── ably-node.js
│       ├── ably.noencryption.js
│       ├── ably.noencryption.min.js
│       └── ably-reactnative.js
├── resources
│   ├── Browserstack-logo@2x.png
│   └── typescript-demo.gif
├── ably.d.ts
├── callbacks.d.ts
├── callbacks.js
├── CHANGELOG.md
├── LICENSE
├── package.json
├── promises.d.ts
├── promises.js
└── README.md
```

```text
❯ jq .version package.json
"1.2.7"
```

```text
❯ cat promises.js | head -n 10
"use strict";
function promisifyOptions(options) {
  if(typeof options == 'string') {
    options = (options.indexOf(':') == -1) ? {token: options} : {key: options};
  }
  options.promises = true;
  return options;
}

var Ably = require('./nodejs/index');
```

┆Issue is synchronized with this [Jira Uncategorised](https://ably.atlassian.net/browse/SDK-676) by [Unito](https://www.unito.io/learn-more)

--- a/callbacks.js
+++ b/callbacks.js
@@ -1,4 +1,4 @@
 "use strict";
 /* When returning promises becomes the default in some future version, this
  * file will start modifying the exports as promises.js does today */
-module.exports = require('./nodejs/index');
+module.exports = require('./browser/static/ably-node');

--- a/callbacks.js
+++ b/callbacks.js
@@ -1,4 +1,6 @@
 "use strict";
 /* When returning promises becomes the default in some future version, this
- * file will start modifying the exports as promises.js does today */
+ * file will start modifying the exports as promises.js does today.
+ * Please note that the file imported below is only generated after running 
+ * the build task. */
 module.exports = require('./browser/static/ably-node');

--- a/nodejs/index.js
+++ b/nodejs/index.js
@@ -1,2 +1,0 @@
-"use strict";
-module.exports = require('../browser/static/ably-node');

--- a/package.json
+++ b/package.json
@@ -8,7 +8,7 @@
     "email": "support@ably.io"
   },
   "main": "./browser/static/ably-node.js",
-  "typings": "./browser/static/ably.d.ts",
+  "typings": "./ably.d.ts",
   "react-native": {
     "./browser/static/ably-node.js": "./browser/static/ably-reactnative.js"
   },

--- a/promises.js
+++ b/promises.js
@@ -7,6 +7,8 @@ function promisifyOptions(options) {
   return options;
 }
 
+/* Please note that the file imported below is only generated after running 
+ * the build task. */
 var Ably = require('./browser/static/ably-node');
 
 var RestPromise = function(options) {

--- a/promises.js
+++ b/promises.js
@@ -7,7 +7,7 @@ function promisifyOptions(options) {
   return options;
 }
 
-var Ably = require('./nodejs/index');
+var Ably = require('./browser/static/ably-node');
 
 var RestPromise = function(options) {
   return new Ably.Rest(promisifyOptions(options));

